txnkv/txnsnapshot/snapshot.go

```diff
@@ -903,6 +903,7 @@ func (s *KVSnapshot) SetRPCInterceptor(it interceptor.RPCInterceptor) {
 }
 
 // AddRPCInterceptor adds an interceptor, the order of addition is the order of execution.
+// the chained interceptors will be dedupcated by its name.
```

Comment:
Why need this?

Reply:
Currently, the same interceptor is set in different places in tidb, I'm not sure if they are set several because the current behavior is replace. But I need to change the behavior to add